### PR TITLE
fix(module:tree-select): fix search box exception when Chinese search

### PR DESCRIPTION
**Pull request by EnochGao**
fix bug: #8165

### CODE REVIEW

**Comment by Nicoss54** [comment]
Hi @EnochGao thanks a lot fot this PR, could you please provide the tests for the code added :) 

Thanks in advance :)

**Review by Nicoss54 — CHANGES_REQUESTED** [review]
Hi @EnochGao thanks a lot fot this PR, could you please provide the tests for the code added :)

Thanks in advance :)

**Comment by EnochGao** [comment]
> Hi @EnochGao thanks a lot fot this PR, could you please provide the tests for the code added :)
> 
> Thanks in advance :)

Kindly verify it

File: components/tree-select/tree-select.spec.ts [hunk 1]
```diff
@@ -22,6 +31,38 @@ import { NzFormControlStatusType, NzFormModule } from 'ng-zorro-antd/form';
 import { NzTreeSelectComponent } from './tree-select.component';
 import { NzTreeSelectModule } from './tree-select.module';
 
+describe('nz-tree-select component', () => {
```
Comment (Nicoss54) [inline components/tree-select/tree-select.spec.ts:34]: suggestion: all the test are correct, but please move them into the basics section please

**Review by Nicoss54 — APPROVED** [review]

**Comment by Nicoss54** [comment]
LGTM

**Pull request merged**